stubs/redis/redis/exceptions.pyi

```diff
@@ -14,4 +14,9 @@ class WatchError(RedisError): ...
 class NoScriptError(ResponseError): ...
 class ExecAbortError(ResponseError): ...
 class ReadOnlyError(ResponseError): ...
+class NoPermissionError(ResponseError): ...
+class ModuleError(ResponseError): ...
```

Comment:
I don't have `ModuleError`. Do I have the wrong version of redis installed?

```
$ . redisenv/bin/activate
(redisenv) $ pip freeze |grep redis
redis==3.5.3
(redisenv) $ grep ModuleError redisenv/lib/python3.7/site-packages/redis/exceptions.py 
(redisenv) $
```

Reply:
Oh, good point. I don't think that change has released. I was reading exceptions.py from master. fixed